### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by schulzad**
lots of fixes some of them we went through.
have a working mock of hub, next pr will fix up cloud only tests to work with mock

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,16 +32,17 @@ update `src/ap_credentials.yaml` with credentials for a test access point
 ### Getting cloud access tokens
 
 To do this is to use [Consigliere](https://consigliere-regionald.smartthingsgdev.com/login#)
```
Comment (pnelsen) [inline README.md]: We might want to change this to `/tokens`

File: README.md [hunk 2]
```diff
@@ -17,12 +17,11 @@ Note: This is a __python3__ only project
...
-  1. mobile token (see #Getting cloud access tokens])
-  2. serial number
-
+  1.  mobile token (see #[Getting cloud access tokens])
+  2.  serial number
 3. `cd src`
-3. `python add_hub.py`
-4. this will gennerate certs and other data in `gen.claim.common.yaml`
+4. `python add_hub.py`
+5. this will gennerate certs and other data in `gen.claim.common.yaml`
```
Comment (pnelsen) [inline README.md]: nit: generate

File: README.md (hunk 1 above)
Comment (krupatil) [reply README.md]: Yes, I faced the same issue. It should be https://account.smartthings.com/tokens

File: src/test_e2e.wifi.tavern.yaml [hunk 3]
```diff
@@ -100,8 +88,7 @@ stages:
         reason: "NONE"
 
 
-
-  - name: HACK Disconnect to AP via bad credentials
+  - name: HACK Disconnect to AP by sending bad credentials of a known AP
```
Comment (jmagnuson) [inline src/test_e2e.wifi.tavern.yaml]: Can the `psk` be "badsecret" or something? In some cases, wpa_supplicant crashes out with psk's shorter than 8 characters.

**Pull request merged**